cmd/atlas/internal/cmdapi/migrate_test.go

```diff
@@ -11,6 +11,7 @@ import (
 	"encoding/json"
 	"errors"
 	"fmt"
+	"github.com/stretchr/testify/assert"
```

Comment:
```suggestion
```
Use "require" instead. I prefer the "fail fast" approach, and that's why "require" is the main used here.